pkg/model/operations/op.go

```diff
@@ -20,23 +20,23 @@ import (
 	"github.com/gertd/go-pluralize"
 )
 
-type OpType int
+type OpType string
 
 const (
-	OpTypeUnknown OpType = iota
-	OpTypeCreate
-	OpTypeCreateBatch
-	OpTypeDelete
-	OpTypeReplace
-	OpTypeUpdate
-	OpTypeAddChild
-	OpTypeAddChildren
-	OpTypeRemoveChild
-	OpTypeRemoveChildren
-	OpTypeGet
-	OpTypeList
-	OpTypeGetAttributes
-	OpTypeSetAttributes
+	OpTypeUnknown OpType = "Unknown"
+	OpTypeCreate = "Create"
+	OpTypeCreateBatch = "CreateBatch"
+	OpTypeDelete = "Delete"
+	OpTypeReplace = "Replace"
+	OpTypeUpdate = "Update"
+	OpTypeAddChild = "AddChild"
+	OpTypeAddChildren = "AddChildren"
+	OpTypeRemoveChild = "RemoveChild"
+	OpTypeRemoveChildren = "RemoveChildren"
+	OpTypeGet = "Get"
+	OpTypeList = "List"
+	OpTypeGetAttributes = "GetAttributes"
+	OpTypeSetAttributes = "SetAttributes"
```

Comment:
I'm a little confused about this... any reason for it?

Reply:
This was originally to allow us to go straight from config JSON to an `OpType` but as discussed below this has been reverted to an `int`.